src/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from './private/index';
 export * from './public/index';
+export { meeting } from './public/meeting';
```

Comment:
I think this needs to go into the './public/index.ts' file instead. Can you please move it there?

Reply:
You can delete this line now

Reply:
Deleted